tests/func/test_metrics.py

```diff
@@ -279,6 +279,20 @@ def test_formatted_output(self):
         assert expected_txt in stdout
         assert expected_json in stdout
 
+    def test_show_all_should_be_current_dir_agnostic(self):
+
+        metric_file = "mfile"
+
+        ret = main(
+            ["run", "-m", metric_file, "echo 0.5 >> {}".format(metric_file)]
+        )
+        self.assertEqual(0, ret)
+
+        os.chdir(self.DATA_DIR)
+
+        ret = main(["metrics", "show", "-a"])
```

Comment:
We would expect that it would print relpath, right? If so, should we check it too?